core/src/test/resources/sql/higher-order-functions.iq

```diff
@@ -0,0 +1,90 @@
+# higher-order-functions.iq - Queries involving higher-order functions
```

Comment:
Can you rename this file to `lambda.iq`?

It's debatable whether SQL lambdas are true higher-order functions.